tests/test_bootstrap_template_mirror.sh

```
@@ -0,0 +1,636 @@
+  . "$ROOT/scripts/bootstrap/template-mirror.sh"
+  # The libs set `set -euo pipefail`; defeat -e here so we can
+  # capture the helper rc into a variable and echo it. -u + pipefail
+  # are still desirable safety nets but -e would exit the subshell
+  # before reaching the echo on a non-zero rc.
+  set +e
+  BOOTSTRAP_DRY_RUN=0
+  BOOTSTRAP_LOG_FILE=""
+  bootstrap::_clean_repo_template_yml "$TARGET"
+  echo "RC=$?"
+' 2>&1)
+no_yq_ec=$?
```

Shellcheck (0.11.0)

[warning] no_yq_ec appears unused. Verify use (or export if used externally).

(SC2034)